src/hooks/worlds/useOwnWorlds.ts

```diff
@@ -0,0 +1,36 @@
+import { useFirestore, useFirestoreCollectionData } from "reactfire";
+import { collection, query, where } from "firebase/firestore";
+
+import { ALWAYS_EMPTY_ARRAY, COLLECTION_WORLDS } from "settings";
+
+import { World } from "api/world";
+
+import { CONVERTER_WORLD_WITH_ID } from "utils/converters";
+import { WithId } from "utils/id";
+
+import { useUser } from "hooks/useUser";
+
+export const useOwnWorlds = (): {
```

Comment:
Maybe extract the type and add the return like:

```ts
type UseOwnWorlds = () => LoadStatus & { ownWorlds: WorldWithId[]; };

export const useOwnWorlds: UseOwnWorlds = () => {
```

The `LoadStatus` allows for some general hook interface applicable to components that might check positive or negative loading.